app/controllers/admin/playlists_controller.rb

```diff
@@ -7,5 +7,13 @@ def authorize_resource
       @resource = Student.find(params[:student_id]).playlist
       authorize(resource)
     end
+
+    def collection
+      Playlist.includes(:student)
+              .where(users: {grove_id: current_user.grove.id})
```

Comment:
this should be shorted to `where(users: {grove: current_user.grove})` or `where(users: {grove_id: current_user.grove_id})` lets not use objects where values will do.